src/test/url-handling.test.js

```diff
@@ -178,6 +183,20 @@ describe('url upgrading', function() {
       expect(urlStateReducers.getSelectedTab(getState())).toBe('markers-table');
     });
   });
+
+  describe('general checks', function() {
+    it("won't run if the version is specified", function() {
+      const { getState } = _getStoreWithURL({
+        pathname: '/public/e71ce9584da34298627fb66ac7f2f245ba5edbf5/markers/',
+        search: `?v=${CURRENT_URL_VERSION}`,
+      });
+
+      // The conversion process shouldn't run
```

Comment:
The value you are asserting here is a bit magical. Is there any way to explain for future readers why it's there?

Reply:
yeah, it's not perfect, I can add a comment explaining this. With my current WIP patch it would be easier (as I reuse a parameter so it's much more visible) so I might change this test along with that patch.